rasa/utils/plotting.py

```diff
@@ -133,21 +133,36 @@ def plot_histogram(
     # Wine-ish colour for the confidences of hits.
     # Blue-ish colour for the confidences of misses.
     colors = ["#009292", "#920000"]
-    bins = [0.05 * i for i in range(1, 21)]
+    n_bins = 25
+    max_value = max(
+        [max(hist_data[0], default=0), max(hist_data[1], default=0)], default=0
+    )
```

Comment:
@Ghostvv I tried another strategy on sara's data, how about these - 
`cosine`(left), `inner`(right)
![intent_histogram](https://user-images.githubusercontent.com/8708249/107154726-96c05800-699a-11eb-9a1f-745ed2fea630.png)
![intent_histogram](https://user-images.githubusercontent.com/8708249/107154762-b9527100-699a-11eb-9de3-494df273cf13.png)

Reply:
Alternatively, if we feel the bar thickness is distracting we could keep the plotting strategy as before for `softmax` and `cosine` because most users will use those two options and use the above one for `inner`.